apps/studio/src/server/modules/resource/resource.service.ts

```diff
@@ -53,7 +53,7 @@ export const updatePageById = (
   })
 }
 
-export const updateBlobById = (props: { id: number; content: Page }) => {
+export const updateBlobById = (props: { id: number; content: string }) => {
```

Comment:
changed because `content` (of the blob) should be opaque - this is string because we treat it as json